libs/tup-components/src/projects/ProjectsListing/ProjectsListing.module.css

```diff
@@ -35,20 +42,15 @@
 
 /* Project summary */
 .project-title {
-    font-weight: 500;
+    font-weight: var(--bold);
+    font-size: var(--global-font-size--medium);
 }
 .project-pi {
     background: #CAFCC8;
     width: fit-content;
     padding: 3px;
     margin-left: -3px;
 }
-.project-summary-separator {
-    border-bottom: 1px solid #C7C7C7;
-    width: 100%;
-    margin-top: 10px;
-    margin-bottom: 10px;
-}
```

Comment:
**Explanation:** I replaced this with an `<hr />` (and fixed `<hr />` spacing like I had to do on CMS).